libraries/common/subscriptions/helpers/handleLinks.js

```diff
@@ -143,11 +137,35 @@ export const isShopLink = (location) => {
   return true;
 };
 
+/**
+ * Check if the given pathname is a protector route.
+ * @param {string} location The location to check.
+ * @return {boolean}
+ */
+export const isProtector = location => Array.from(authRoutes.getAll().values()).includes(location);
+
+/**
+ * Finalizes the navigation to a legacy page or the in-app-browser.
```

Comment:
What does that mean? To me, this doesn't make any sense. What is "Finalized" in this case? Please describe more what is it does and why.